pkg/virtctl/vm/vm.go

```diff
@@ -289,6 +321,14 @@ func usage(cmd string) string {
 	return usage
 }
 
+func usageExpand(cmd string) string {
```

Comment:
Inline the `cmd` variable and avoid the calls to `fmt.Sprintf`? Define the multiline string like in `usageAddVolume` below?